app/templates/items/detail/tabs/_item_options.hbs

```diff
@@ -0,0 +1,130 @@
+          <div class="line text-holder">{{ item.inventoryNumber }}</div>
+        </div>
+      </div>
+
+      <!-- LOCATION -->
+      <div class="columns small-3 sub-block main">
+        <div class="text-block">
+          <div class="line icon-holder">{{fa-icon 'map-marker-alt' size="lg"}}</div>
+          <div class="line text-holder">
+            {{#if item.hasMultiLocations}}
+            {{t "item_status_bar.multiple"}}
+            {{else}}
+            {{#if item.firstAllLocationName}}
+            {{item.firstAllLocationName}}
+            {{else}}
+            N/A
+            {{/if}}
+            {{/if}}
+          </div>
+        </div>
+      </div>
+
+      <div class="columns small-6 tab sub-block main">
+        <div class="text-block row divided">
+          <div class="subtab column small-4">
+            <div class="line icon-holder">{{fa-icon 'paper-plane' size="lg"}}</div>
+            <div class="line text-holder"> {{ item.totalDispatchedQty }} </div>
+          </div>
+          <div class="subtab column small-4">
+            <div class="line icon-holder">{{fa-icon 'shopping-basket' size="lg"}}</div>
+            <div class="line text-holder"> {{ item.totalDesignatedQty }} </div>
+          </div>
+          <div class="subtab column small-4">
+            <div class="line icon-holder">{{fa-icon (if item.allowWebPublish 'eye' 'eye-slash') size="lg"}}</div>
+            <div class="line text-holder"> {{ item.availableQty }} </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="order-preview">
+      <div class="row gc-orders-package-block">
+        <div class="columns small-3 large-2 left">
+          <img src={{item.thumbImageUrl}} class="thumb" />
+        </div>
+
+        <div class="columns small-9 large-10 sub-block main" >
+          <div class="text-block">
+            <div class="line">{{ item.code.name }}</div>
+            <div class="line">{{ item.notes }}</div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="form-container">
+      <div class="title">{{fa-icon actionIcon}} {{actionName}}</div>
+      <div class="infos">
+        <div class="info-row">
+          <div class="key">Item</div>
+          <div class="value">{{fa-icon 'tag'}} {{item.inventoryNumber}}</div>
+        </div>
+
+        <div class="info-row">
+          <div class="key">From Location</div>
+          <div class="value clickable" {{action "selectLocationAction"}}>{{fa-icon 'map-marker-alt'}}
+            {{actionFrom.displayName}}</div>
+        </div>
+
+        <div class="info-row">
+          <div class="key">Quantity</div>
+          <div class="value">
+            {{input type="number" value=actionQty class="qty-input"}}
+          </div>
+          <div class="sub-text {{if (is-not (is-in-range actionQty 1 maxQuantity)) 'warn-text'}}">Max: {{maxQuantity}}
```

Comment:
I think these logics can be wrapped inside `let-alias` helper. It will become more readable.
```
{{#let-alias if (is-not (is-in-range actionQty 1 maxQuantity)) as |isMax| }}
      <div class="sub-text {{isMax  'warn-text'}}">
{{/let-alias
```
Just a suggestion.